store.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Karma</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <nav>
+        <img
+          src="./img/karma-logo.svg"
+          alt="karma-logo"
+          class="logo"
+          width="30px"
+        />
+        <ul class="nav-links">
+          <li>
+            <a href="index.html"><strong>Meet Karma</strong></a>
+          </li>
+          <li><a href="#how-it-works">How it Works</a></li>
+          <li><a class="store" href="store.html">Store</a></li>
+          <li><a href="#blog">Blog</a></li>
+          <li><a href="#help">Help</a></li>
+          <li><a href="#login">Login</a></li>
+        </ul>
+      </nav>
+  <body>
+    <main>
+      <div class="wrapper">
+      
+<!-- <div class="form-wrapper">
+  <div class="form-container"> -->
+    <form>
+        <h2>Order your Karma wifi device today!</h2>
+        <div class="name">
+          <div class = "first-last">
+        <label for="first-name">First name *</label>
+        <input type="text" id="name" name="name" required> 
+          </div>
+          <div class="first-last" >
+        <label for="last-name">Last name *</label>
+        <input type="text" id="last-name" name="last-name" required>
+        </div>
+        </div>
+        <label class="address" for="address">Address *</label>
+        <input type="text" id="address" name="address" required>
+       
+       
+        <label class="address2" for="address2">Address 2 *</label>
+        <input type="text" id="address2" name="address2" required>
+
+       <div class="address-info">
+        <div class="city-postcode">
+        <label for="city">City</label>
+        <select id="city" >
+            <option value ="Select-your-city...">Select your city...</option>
```

Comment:
You can add cities to your drop down by adding `<option value="London">London</option>` as children to the <select/> tag. 

Additionally, if you want to have a "placeholder", one tricky way to do that with a option tags is to add 
`<option disabled="" selected="">Select City...</option>`. Don't be afraid to google thesee types of questions: [Placeholder for select tag]( https://stackoverflow.com/questions/17603055/placeholder-for-select-tag)